doc/releasenotes.html

```diff
@@ -59,6 +59,55 @@ <h2>CATEGORY: General, Solar, Detailed Photovoltaic, Wind Power, CSP Physical Tr
 
 -->
 
+<h1 id="2023-12-17-r1">SAM 2023.12.17 r1, SSC 28x: March xx, 2024</h1>
+
+<p>This is the first maintenance update to <a href="#2023-12-17">SAM 2023.12.17</a>.</p>
+
+<ul>
+<li>Solar Resource
```

Comment:
Additional solar resource issue that was fixed: https://github.com/NREL/SAM/pull/1710